org.gnome.EasyTAG.yaml

```diff
@@ -0,0 +1,122 @@
+id: org.gnome.EasyTAG
+runtime: org.gnome.Platform
+runtime-version: 3.28
```

Comment:
This really should use `3.38`. intltool is available in shared-modules: https://github.com/flathub/shared-modules/blob/c1fec5caead73848a2fdb2a982f3760d1a4e2604/intltool/intltool-0.51.json

Reply:
Fixed in commit e2a92ec, although I did end up adding a patch to the appdata.xml bundled with the source tarball so it passes the validation (it's still using the external appdata.xml in the end).